src/main/java/marquez/common/models/Namespace.java

```diff
@@ -0,0 +1,29 @@
+package marquez.common.models;
+
+import java.util.regex.Pattern;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+
+@EqualsAndHashCode
+@ToString
+public final class Namespace {
+  private static final Pattern NAMESPACE_PATTERN = Pattern.compile("^[a-zA-Z0-9_]{1,1024}$");
```

Comment:
same here re: the `1024`, eventually want to pull it out

Reply:
👍